app/liquid_tags/liquid_tag_base.rb

```diff
@@ -19,6 +19,12 @@ def strip_tags(string)
     ActionController::Base.helpers.strip_tags(string).strip
   end
 
+  def pattern_match_for(input, regex_options)
```

Comment:
I have used this method in two liquid tags now, so I decided to abstract it, as I will probably use it more often.